java/code/src/com/redhat/rhn/frontend/action/CSVDownloadAction.java

```diff
@@ -113,7 +113,7 @@ protected String getExportColumns(HttpServletRequest request, HttpSession sessio
      * @param session HTTP session
      * @return page data
      */
-    protected List<BaseDto> getPageData(HttpServletRequest request, HttpSession session) {
+    protected List<Object> getPageData(HttpServletRequest request, HttpSession session) {
```

Comment:
Why changing from `BaseDto` to `Object` here? If we return anything different than a `BaseDto` we're likely to have troubles somewhere else.